python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -734,6 +741,7 @@ def _run_task_kwargs(
                     ephemeral_storage=container_context.run_resources.get("ephemeral_storage"),
                     volumes=container_context.volumes,
                     mount_points=container_context.mount_points,
+                    readonly_root_filesystem=container_context.readonly_root_filesystem,
```

Comment:
once this is just on the run launcher and not on EcsContainerContext, this can be `self.readonly_root_filesystem` instead